package-lock.json

```diff
@@ -1,3187 +1,3158 @@
 {
   "name": "@devoinc/browser-sdk",
-  "version": "2.0.0",
-  "lockfileVersion": 2,
+  "version": "3.0.0",
+  "lockfileVersion": 1,
```

Comment:
Wrong file version here. You must have updated/recreated the lock with a version of npm < 6.